x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/search_ui.tsx

```diff
@@ -31,7 +43,51 @@ export const SearchUI: React.FC = () => {
       <SetPageChrome trail={getEngineBreadcrumbs([SEARCH_UI_TITLE])} />
       <EuiPageHeader pageTitle={SEARCH_UI_TITLE} />
       <FlashMessages />
-      <EuiPageContentBody>TODO</EuiPageContentBody>
+      <EuiPageContentBody>
+        <EuiFlexGroup alignItems="flexStart">
+          <EuiFlexItem>
+            <EuiText size="s" color="subdued">
+              <p>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.appSearch.engine.searchUI.bodyDescription"
+                  defaultMessage="Search UI is a free and open library for building search experiences with React. {link}."
```

Comment:
[Super optional] Since these are separate clauses/sentences I think you could probably get away with keeping them as separate / non-nested i18n sections. e.g.,

```js
<FormattedMessage
  id="xpack.enterpriseSearch.appSearch.engine.searchUI.bodyDescription"
  defaultMessage="Search UI is a free and open library for building search experiences with React."
/>
<EuiLink ...>
  {i18n.translate('...', { defaultMessage: 'Learn more.' })}
</EuiLink>
```

Not a huge deal by any means though and you've correctly translated the nested link text, so I'm good either way!